src/run/runner/helpers/run_command_with_log_pipe.rs

```diff
@@ -46,5 +49,13 @@ pub fn run_command_with_log_pipe(mut cmd: Command) -> Result<ExitStatus> {
     thread::spawn(move || {
         log_tee(stderr, std::io::stderr(), Some("[stderr]")).unwrap();
     });
+
+    let perf_pid = process.id();
```

Comment:
this is not necessarily a perf_pid here, this is just the argument command's pid